cmd2/utils.py

```diff
@@ -477,11 +477,7 @@ def write(self, s: str) -> None:
         """
         if not isinstance(s, str):
             raise TypeError('write() argument must be str, not {}'.format(type(s)))
-
-        if not self.pause_storage:
-            self.buffer.byte_buf += s.encode(encoding=self.encoding, errors=self.errors)
-        if self.echo:
-            self.inner_stream.write(s)
+        self.buffer.write(s.encode(self.encoding, errors=self.errors))
```

Comment:
This approach replaces `inner_stream.write()` with `inner_stream.buffer.write()`. Does writing directly to the buffer remove support for newline translation and unbuffered writes?

```Python
class io.TextIOWrapper(buffer, encoding=None, errors=None, newline=None,
                       line_buffering=False, write_through=False)
```

Reply:
I don't know how to even check that. Can you think of a way a unit test could verify this?